### PR TITLE
AKS Baseline regulated - Bug fixes  - Ingress-controller certgen image regression issue - CA mismatch  - osm certificate rotation

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: cluster-manifests/ingress-nginx/deployment.yaml [hunk 1]
```diff
@@ -302,12 +303,12 @@ spec:
...
             - --election-id=ingress-controller-leader
             - --controller-class=k8s.io/ingress-nginx
+            - --ingress-class=nginx
             - --default-ssl-certificate=$(POD_NAMESPACE)/ingress-tls-csi
             - --configmap=$(POD_NAMESPACE)/ingress-nginx-controller
             - --validating-webhook=:8443
             - --validating-webhook-certificate=/usr/local/certificates/cert
             - --validating-webhook-key=/usr/local/certificates/key
-            - --v=3
           securityContext:
```
Comment (ckittel) [inline cluster-manifests/ingress-nginx/deployment.yaml:312]: Looks like we dropped the added logging.  Can we add this back in for observability?

**Review by ckittel — APPROVED** [review]
Besides this one item, LGTM 🚀  (I know you're going to hold off for ~30 hours to make sure the cert rotates fine.)

File: cluster-manifests/ingress-nginx/deployment.yaml (hunk 1 above)
Comment (magrande) [reply cluster-manifests/ingress-nginx/deployment.yaml]: Good catch ! added

Comment (ckittel) [reply cluster-manifests/ingress-nginx/deployment.yaml]: Thanks!

**Comment by magrande** [comment]
Tests went pretty well ! Merging !

**Pull request merged**